ocs_ci/resiliency/resiliency_workload_config.py

```diff
@@ -0,0 +1,331 @@
+"""
+Configuration loader for resiliency workload testing.
+
+This module provides a configuration management system for resiliency tests,
+similar to the KrknWorkloadConfig used in krkn chaos tests.
+"""
+
+import logging
+from typing import Dict, Any, List
+
+from ocs_ci.framework import config
+
+log = logging.getLogger(__name__)
+
+
+class ResiliencyWorkloadConfig:
+    """
+    Configuration loader for resiliency workload testing.
+
+    This class accesses configuration from the framework's config system
+    and provides methods to access workload-specific settings.
+
+    Configuration should be passed via --ocsci-conf parameter:
+        pytest --ocsci-conf conf/ocsci/resiliency_tests_config.yaml ...
+
+    The framework loads and merges all configs during initialization,
+    so by the time this class is instantiated, config.ENV_DATA already
+    contains the merged configuration.
+    """
+
+    def __init__(self):
+        """
+        Initialize ResiliencyWorkloadConfig.
+        Uses the framework's global config object.
+
+        Note: Config should already be loaded via --ocsci-conf parameter.
+        This class just provides convenience methods to access resiliency_config.
+        """
+        self.config = config
+
+    def get_workloads(self) -> List[str]:
+        """
+        Get list of enabled workload types.
+
+        Returns:
+            list: List of workload type strings
+        """
+        resiliency_config = self.config.ENV_DATA.get("resiliency_config", {})
+        workloads = resiliency_config.get("workloads", ["VDBENCH"])
```

Comment:
Maybe if no workloads specify we should fail the test and not go with VDBENCH. Could be that the user did some typo mistake like:
worloads:
  - RGW
And no one will know of as it will run with VDBENCH.